doc/api/buffer.md

```diff
@@ -2010,7 +2010,14 @@ const buf = Buffer.alloc(256);
 const len = buf.write('\u00bd + \u00bc = \u00be', 0);
 
 console.log(`${len} bytes: ${buf.toString('utf8', 0, len)}`);
-// Prints: 12 bytes: ½ + ¼ = ¾
+// Prints: 12 bytes: ½ + ¼ ¾
```

Comment:
The output includes the `=` sign, though.

Reply:
Mistakenly edited file in two tabs. Got deleted.